### PR TITLE
Bump version to v0.2.0

### PR DESCRIPTION
**Pull request by manifest**

### CODE REVIEW

File: src/mqtt.rs [hunk 1]
```diff
@@ -94,9 +94,7 @@ impl AgentBuilder {
             .port_part()
             .ok_or_else(|| err_msg("missing MQTT port"))?;
 
-        Ok(rumqtt::MqttOptions::new(client_id, host, port.as_u16())
-            .set_keep_alive(30)
-            .set_reconnect_opts(rumqtt::ReconnectOptions::AfterFirstSuccess(5)))
+        Ok(rumqtt::MqttOptions::new(client_id, host, port.as_u16()))
```
Comment (0nkery) [inline src/mqtt.rs]: Why did you remove `ReconnectOptions`?

File: src/mqtt.rs [hunk 2]
```diff
@@ -324,20 +315,20 @@ pub struct OutgoingRequestProperties {
...
+            response_topic: response_topic.to_owned(),
+            correlation_data: correlation_data.to_owned(),
+            authn: None,
         }
     }
 
+    pub fn set_authn(&mut self, authn: AuthnProperties) -> &mut Self {
+        self.authn = Some(authn);
+        self
+    }
```
Comment (0nkery) [inline src/mqtt.rs]: Is it okay with borrow checker? I guess there will be dragons.

File: src/mqtt.rs (hunk 1 above)
Comment (0nkery) [reply src/mqtt.rs]: Okay, I see changes in `rumqtt`.

File: src/mqtt.rs (hunk 2 above)
Comment (manifest) [reply src/mqtt.rs]: That is ok.

File: src/lib.rs [hunk 3]
```diff
@@ -136,38 +132,44 @@ pub enum Destination {
...
-    ) -> EventSubscription<'a> {
-        EventSubscription::new(Source::Broadcast(from, uri))
+    pub fn broadcast_events<'a, A>(from: &'a A, uri: &'a str) -> EventSubscription<'a>
+    where
+        A: Authenticable,
+    {
+        EventSubscription::new(Source::Broadcast(from.as_account_id(), uri))
     }
 
     pub fn multicast_requests<'a>() -> RequestSubscription<'a> {
```
Comment (0nkery) [inline src/lib.rs]: `Multicast` should have `AccountId` too, I guess?

Comment (manifest) [reply src/lib.rs]: For what reason?

Comment (0nkery) [reply src/lib.rs]: Sorry, I thought it was a `Destination`. :)
